centos/centos-7.x/common/install_amd_libs.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+yum install -y texinfo
```

Comment:
can we move this line into the install_utils.sh? That way you dont have to keep separate install_amd_libs.sh